google/cloud/firestore_v1/query.py

```diff
@@ -333,16 +339,17 @@ def order_by(self, field_path, direction=ASCENDING):
             field_filters=self._field_filters,
             orders=new_orders,
             limit=self._limit,
+            limit_to_last=self._limit_to_last,
             offset=self._offset,
             start_at=self._start_at,
             end_at=self._end_at,
             all_descendants=self._all_descendants,
         )
 
     def limit(self, count):
-        """Limit a query to return a fixed number of results.
+        """Limit a query to return the next `count` matching results.
 
-        If the current query already has a limit set, this will overwrite it.
+        If the current query already has a `limit` set, this will override it.
```

Comment:
Can we add a note here stating that `limit` and `limit_to_last` are mutually exclusive?

i.e. this: `query.limit_to_last(10).limit(5)` doesn't really make sense. The result will end up being the same as `query.limit(5)`

Reply:
Done